tests/Functional/Action/Lti/GetUserAssignmentLtiLinkActionTest.php

```diff
@@ -72,7 +72,10 @@ public function testItReturns405IfHttpMethodIsNotAllowed(): void
     {
         $this->kernelBrowser->request('POST', '/api/v1/assignments/1/lti-link');
 
-        self::assertSame(Response::HTTP_METHOD_NOT_ALLOWED, $this->kernelBrowser->getResponse()->getStatusCode());
```

Comment:
This was fitting on the line size, why change it? **Same for others**

Reply:
Actually, it was more than the PSR12 spec. which is 120. That was why I changed it.

Reply:
Missed that, thought is was fitting the size.